components/admin_console/admin_settings.jsx

```diff
@@ -11,7 +11,7 @@ import SaveButton from 'components/save_button.jsx';
 import FormError from 'components/form_error.jsx';
 import Constants from 'utils/constants.jsx';
 
-import AdminHeader from 'components/widgets/admin_console/admin_header.jsx';
```

Comment:
OMG yes. I really didn't like the fact that we were specifying the extensions in the import :)